UI/window-projector.cpp

```diff
@@ -8,6 +8,7 @@
 #include "display-helpers.hpp"
 #include "qt-wrappers.hpp"
 #include "platform.hpp"
+#include <QtWidgets>
```

Comment:
This doesn't appear to be required to build?

Reply:
This might have been a leftover from initial work